src/frontend/render/table.rs

```diff
@@ -118,7 +120,38 @@ fn render_table_row(
         .bottom_margin(0)
         .style(Style::default().fg(row_color))
 }
-
+///Returns a Cell matched on short char of the Column
```

Comment:
Nit: 1 blank line between functions and 1 pace at the start of a comment

Reply:
Noted